### PR TITLE
[SHARE-1021][Improvement] Add type mapping for University of Cambridge

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose 
University of Cambridge asked us to map some types.

## Changes
* Add type mapping for University of Cambridge
* Update CHANGELOG

## Release steps
Run `./manage.py loadsources uk.cambridge --overwrite` after merging on respective environments.

### CODE REVIEW

**Comment by aaxelb** [comment]
@laurenbarker, yeah, that makes sense 👍

File: share/sources/uk.cambridge/source.yaml [hunk 1]
```diff
@@ -13,7 +13,12 @@ configs:
     approved_sets: null
     emitted_type: CreativeWork
     property_list: []
-    type_map: {}
+    type_map: {
```
Comment (aaxelb) [inline share/sources/uk.cambridge/source.yaml]: Minorest nitpick: You don't need these braces in yaml

File: share/sources/uk.cambridge/source.yaml [hunk 2]
```diff
@@ -13,7 +13,11 @@ configs:
     approved_sets: null
     emitted_type: CreativeWork
     property_list: []
-    type_map: {}
+    type_map:
+      Journal Article: article,
+      Book chapter: book,
+      Book or Book Chapter: book,
+      Conference Object: conferencepaper,
```
Comment (aaxelb) [inline share/sources/uk.cambridge/source.yaml]: YAML's kinda weird; these trailing commas are actually part of the string.

Comment (laurenbarker) [reply share/sources/uk.cambridge/source.yaml]: Oops. This worked fine though?

Comment (aaxelb) [reply share/sources/uk.cambridge/source.yaml]: Was that when you had the brackets in? Apparently this:
```yaml
foo: {
  apples: bananas,
  potatoes: toes,
}
```
sees the commas as delimiters, but this
```yaml
foo:
  apples: bananas,
  potatoes: toes,
```
sees the commas as part of the strings.

File: share/sources/uk.cambridge/source.yaml [hunk 3]
```diff
@@ -13,7 +13,11 @@ configs:
     approved_sets: null
     emitted_type: CreativeWork
     property_list: []
-    type_map: {}
+    type_map:
+      Journal Article: article
+      Book chapter: book
```
Comment (aaxelb) [inline share/sources/uk.cambridge/source.yaml]: Inconsistent capitalization annoys me. Oh well.

File: share/sources/uk.cambridge/source.yaml (hunk 2 above)
Comment (laurenbarker) [reply share/sources/uk.cambridge/source.yaml]: Oh possibly. That would explain it

**Pull request merged**